pyperf/tests/test_runner.py

```diff
@@ -249,9 +249,8 @@ def test_calibration_zero(self):
         def time_func(loops):
             return 0
 
-        with self.assertRaises(SystemExit):
-            with tests.capture_stdout() as stdout:
-                runner.bench_time_func('bench', time_func)
+        with self.assertRaises(SystemExit), tests.capture_stdout() as stdout:
```

Comment:
I prefer to write it on two lines for readability. Maybe just split the line on two lines using `\` line terminator.